app-misc/khal/khal-0.10.1-r2.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_6 python3_7 )
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1
+
+DESCRIPTION="A CalDAV based calendar"
+HOMEPAGE="http://lostpackets.de/khal/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+SLOT="0"
+IUSE="test zsh-completion"
```

Comment:
remove test use flag it is handled by `distutils_enable_tests`

Reply:
1caa7f192c20